### PR TITLE
[FLINK-18912][python][docs] Add Python api tutorial under Python GettingStart

### PR DESCRIPTION
**Pull request by hequn8128**
## What is the purpose of the change

This pull request adds Python api tutorial under Python GettingStart. The tutorial that tells users how to set up a Python project, how to write jobs and how to run jobs locally.

## Brief change log

  - Move Try Flink tutorial under Python GettingStart.
  - Add pointers in the original document page.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/try-flink/python_table_api.md [hunk 1]
```diff
@@ -23,161 +23,9 @@ specific language governing permissions and limitations
 under the License.
```
Comment (sjwiesman) [inline docs/try-flink/python_table_api.md:23]: If you are moving the content, please just delete this page and setup a redirect.

File: docs/try-flink/python_table_api.md [hunk 2]
```diff
@@ -23,161 +23,9 @@ specific language governing permissions and limitations
...
 -->
 
-This walkthrough will quickly get you started building a pure Python Flink project.
```
Comment (sjwiesman) [inline docs/try-flink/python_table_api.md:26]: If you're moving the content, please just delete this page and setup a redirect.

Comment (hequn8128) [reply docs/try-flink/python_table_api.md:26]: Hi, the Python DataStream tutorial will also be added to this page(FLINK-18984). I was thinking about keeping this page in `Try Flink` and add both Python Table API tutorial link and Python DataStream API tutorial link in it. 

What do you think?

Comment (sjwiesman) [reply docs/try-flink/python_table_api.md:26]: that's fine then.

**Comment by hequn8128** [comment]
@sjwiesman Thanks a lot for the review. Merging...

**Pull request merged**